src/lib/types/actions/prices.ts

```diff
@@ -0,0 +1,8 @@
+export const UPDATE_BTC_SPOT = "UPDATE_BTC_SPOT";
+
+export interface UpdateBTCSpot {
```

Comment:
That's totally on me but thinking a bit more about it I think I would rather keep the action type and the action itself grouped in `lib/actions`.
What do you think?

Reply:
I'm not sure what the best practice around here is. Either way there are a lot of files to keep open whenever adding actions or action types, so personally I don't mind either way.
I'm pretty certain you have a lot more experience with react/redux than me so I'd say your call here on what will be more convenient long term. I'm happy to update my PR if you prefer to change it.